flutter/lib/src/event_processor/android_platform_exception_event_processor.dart

```diff
@@ -62,12 +64,41 @@ class AndroidPlatformExceptionEventProcessor implements EventProcessor {
       exception,
     );
 
-    return event.copyWith(
-      exceptions: [
-        ...?exceptions,
-        ...jvmException,
-      ],
-    );
+    final threads = _markDartThreadsAsNonCrashed(event.threads);
+
+    final jvmExceptions = jvmException.map((e) => e.key);
+
+    var jvmThreads = jvmException.map((e) => e.value).toList(growable: false);
+    if (jvmThreads.isNotEmpty) {
+      // filter potential duplicated threads
+      final first = jvmThreads.first;
+      jvmThreads = jvmThreads
+          .skip(1)
+          .where((element) => element.id != first.id)
+          .toList(growable: true);
+      jvmThreads.add(first);
+    }
+
+    return event.copyWith(exceptions: [
+      ...?exceptions,
+      ...jvmExceptions,
+    ], threads: [
+      ...?threads,
+      ...jvmThreads,
+    ]);
+  }
+
+  /// If the crash originated on Android, the Dart side didn't crash.
+  /// Mark it accordingly.
+  List<SentryThread>? _markDartThreadsAsNonCrashed(
+    List<SentryThread>? threads,
+  ) {
+    return threads
+        ?.map((e) => e.copyWith(
+              crashed: false,
+              current: false,
```

Comment:
Should be matched accoringly to https://github.com/getsentry/sentry-dart/pull/853/files#r863674267

Reply:
This should also just mark the actual current isolate as current = true, not others if there are any present

Reply:
Yes, `current` means this thread was being executed or is still alive, and false otherwise, not sure if its even possible to know at runtime, sometimes its better to not set at all rather than assuming a value.
`crashed` means, this is the thread that actually captured the event/exception.